### PR TITLE
Adds docker for local environment

### PR DESCRIPTION
**Pull request by mcarneiro**
Created a Docker image and docker-compose file to speed up local environment setup and updated `readme.md` to reflect it.

## Testing

Follow `readme.md` instructions to run it with docker.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,20 +4,30 @@ Pick colours based on hsl. Scheme-mode with monochromatic, analogic, complement,
...
-Intall Dependencies
+Access it on localhost:3000
 
-```sh
-meteor npm install
-```
+Or manually:
 
-Start Meteor
+1. Intall Meteor
```
Comment (konstrybakov) [inline README.md:17]:
```suggestion
1. Install Meteor
```

File: README.md [hunk 2]
```diff
@@ -4,20 +4,30 @@ Pick colours based on hsl. Scheme-mode with monochromatic, analogic, complement,
...
 
-```sh
-meteor
-```
+    ```sh
+    curl https://install.meteor.com/ | sh
+    ```
+
+2. Intall Dependencies
```
Comment (konstrybakov) [inline README.md:23]:
```suggestion
2. Install Dependencies
```

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+RUN chown 1000:1000 node_modules
+
+USER 1000:1000
+RUN npm install
+
+# Workaround to install meteor distribution in nodes home directory
+RUN meteor help
+
+EXPOSE 3000
+CMD ["meteor"]
```
Comment (konstrybakov) [inline Dockerfile:28]: Add new line here please

File: README.md [hunk 4]
```diff
@@ -4,20 +4,30 @@ Pick colours based on hsl. Scheme-mode with monochromatic, analogic, complement,
 
 ## How to build
 
-Intall Meteor
+With [docker](https://docs.docker.com/get-docker/) and [docker-compose](https://docs.docker.com/compose/install/):
 
 ```sh
-curl https://install.meteor.com/ | sh
+docker-compose up
```
Comment (konstrybakov) [inline README.md:10]: Should we use `docker compose up` as it says:

```sh
$ docker-compose up
Docker Compose is now in the Docker CLI, try `docker compose up`
```

Comment (konstrybakov) [reply README.md:10]: Also what do you think abobut adding the script to package.json

like:

```json
{
  "scripts": {
    "start": "meteor run",
    "start:docker": "docker-compose up"
    ...
  }
}
```

**Comment by konstrybakov** [comment]
Other than the minor suggestions -- looks good! Thank you! ✊✊

**Review by konstrybakov — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (mcarneiro) [reply README.md:10]: Oh, didn't know about it. Will fix it.

Comment (mcarneiro) [reply README.md:10]: I'll just keep `docker-compose` as it's specific for Windows and Mac versions of Docker (not on Linux yet).

**Pull request merged**